.github/workflows/node.js.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
```

Comment:
Useless workflow it's .github repository which contain description of organization.

Reply:
.github repos holds placeholders and organisation's description. However this workflow aims to test node apps. Many repository won't use node. This workflow's place is clearly here.